### ISSUE TITLE
No longer works

### ISSUE THREAD

**Issue by ionosphere80** [GuildEducationInc/aws-jumpcloud#43]
My token mysteriously expired earlier than usual today. I ran `aws-jumpcloud` (version 2.1.5) to renew the token, but it aborted after receiving a 302 from JumpCloud. I upgraded to 2.1.6 and it still fails, but with a different error.

```
Error: JumpCloud's SSO response did not contain the expected
"SAMLResponse" field.
You may have been removed from the Single-Sign On Application for the
profile "redacted", or its URL may be be incorrect. You can check the URL
by visiting the JumpCloud Console in your web browser and confirming
that one of the Single Sign-On Applications has the URL
"https://sso.jumpcloud.com/saml2/redacted". If the URL is correct, aws-
jumpcloud may need to be updated.
```

SSO still works from the JumpCloud console, so I'm wondering if JumpCloud changed their authentication process and broke this utility.

**Comment by qhartman** [comment]
Yeah, we've been tracking this problem today, and that's our assessment as well. The errors we've been seeing mention an unexpected 302.

**Comment by javs-perez** [comment]
Hey @ionosphere80, we saw this issue today. It looks like Jumpcloud changed their URL or response, we are looking into it.

**Comment by ionosphere80** [comment]
I appreciate the quick response. I wonder what else relies on this mechanism.

**Comment by javs-perez** [comment]
Hi @ionosphere80, this should be fixed with version 2.1.7.

**Comment by ionosphere80** [comment]
Works again... appreciate the quick fix!
